pkg/archive/changes_linux.go

```diff
@@ -84,6 +85,22 @@ func walkchunk(path string, fi os.FileInfo, dir string, root *FileInfo) error {
 	}
 	info.stat = stat
 	info.capability, _ = system.Lgetxattr(cpath, "security.capability") // lgetxattr(2): fs access
+	xattrs, err := system.Llistxattr(cpath)
+	if err != nil && err != unix.EOPNOTSUPP {
+		return err
+	}
+	if xattrs != nil {
+		for _, key := range xattrs {
+			if strings.HasPrefix(key, "user.") {
+				if value, _ := system.Lgetxattr(cpath, key); value != nil {
```

Comment:
same here, I don't think we should ignore errors

Reply:
Fixed in 0da4bc60b3babedb098dd7a75ae4ec0b75031eab.